README.md

```diff
@@ -175,6 +175,87 @@ spec:
 When the service is up and running, it will be possible to interact
 with the Glance API and upload an image using the Ceph backend.
 
+## Example: configure Glance with additiona networks
+
+The Glance spec can be used to configure Glance to have the pods
+being attached to additional networks to e.g. connect to a Ceph
+RBD server on a dedicated storage network.
+
+Create a network-attachement-definition whic then can be referenced
+from the Glance API CR.
+
+```
+---
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: storage
+  namespace: openstack
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "storage",
+      "type": "macvlan",
+      "master": "enp7s0.21",
+      "ipam": {
+        "type": "whereabouts",
+        "range": "172.18.0.0/24",
+        "range_start": "172.18.0.50",
+        "range_end": "172.18.0.100"
+      }
+    }
+```
+
+The following represents an example of Glance resource that can be used
+to trigger the service deployment, and have the service pods attached to
+the storage network using the above NetworkAttachmentDefinition.
+
+```
+apiVersion: glance.openstack.org/v1beta1
+kind: Glance
+metadata:
+  name: glance
+spec:
+  ...
+  glanceAPIInternal:
+    ...
+    networkAttachents:
+    - storage
+  glanceAPIExternal:
+    ...
+    networkAttachents:
+    - storage
+...
```

Comment:
Do we want to update the example adding the following:

```
 vip:
   - endpoint: storage
     ip: 172.18.0.202  
     ipAddressPool: storage
     sharedIP: true
```

or something that explain how the CR can be composed?

Reply:
right, will do that